docs/user/discover.asciidoc

```diff
@@ -3,114 +3,227 @@
-<<lucene-query, Lucene query syntax>>, you can access it from the KQL menu.
 
 [float]
-=== Filter your search results
+[[find-the-data-you-want-to-use]]
+=== Step 1. Find the data you want to use
 
-Next, you'll want narrow your search results to a more manageable data set.
-When you click on a name in the field list, you'll see
-the top five values for the field, the number of documents that contain the field,
-and the percentage of documents that contain each value. From this view, you can
-use the (+) magnifier icon to quickly find all
-documents that have that value, or (-) to exclude all
-documents with that value. For more filter options, see <<field-filter, filtering by field>>.
+Tell {kib} where to find your data, then set a time range for your exploration.
 
+. Open the main menu, and select **Discover**.
++
+With **Discover**, you can perform ad-hoc analysis of your data in real-time.
+It’s where you’ll find hidden insights and relationships and emerging trends in your data.
+
+. Select the data you want to work with by choosing an <<index-patterns,index pattern>>.
++
+{kib} requires an index pattern to tell it which Elasticsearch indices you want to work with.
+To view the ecommerce sample data, make sure the index pattern is set to **kibana_sample_data_ecommerce**.
++
 [role="screenshot"]
-image::images/filter-field.png[height=317]
+image::images/discover-index-pattern.png[How to set the index pattern in Discover]
+
+. Adjust the time range to *Last 7 days*.
++
+NOTE: The range selection is based on the default time field in your data.
+If you are using the sample data, this value was set when you added the data.
+If you  are using your own data, and it does not have a time field, the range selection is not available.
 
+. Optional. To view the count of documents for a given time,
+click and drag the mouse over the histogram, and then click *Refresh*.
 
 [float]
-=== Add and remove fields
+[[explore-fields-in-your-data]]
+=== Step 2. Explore the fields in your data
 
-The sortable documents table
-lists the documents that match your search.
+The sortable documents table lists the documents that match your search.
 By default, the table includes columns for the time field and the document `_source`.
-To zero in on a specific field, click *add* next to the field name.
-For example, if you add the `currency`, `customer_last_name`, and `day_of_week` fields,
-the document table includes columns for those three fields.
+You’ll modify this table to display your fields of interest.  For this guide,
+the goal is to dig into the ecommerce data and find fields that
+help determine when and where to target your ads.
+
+. In the side navigation, scan through the list of **Available fields** to see
+what’s in your data. You can also search for a field by name using the
+**Search** field at the top of the list.
+
+. Find the `manufacturer` field, and then click it to view its top five values.
```

Comment:
Why do they want to view the top five values? How does this help them answer their questions?